src/zulip.rs

```diff
@@ -491,20 +491,26 @@ async fn acknowledge(gh_id: i64, mut words: impl Iterator<Item = &str>) -> anyho
     };
     match delete_ping(&mut crate::db::make_client().await?, gh_id, ident).await {
         Ok(deleted) => {
-            let mut resp = format!("Acknowledged:\n");
-            for deleted in deleted {
-                resp.push_str(&format!(
-                    " * [{}]({}){}\n",
-                    deleted
-                        .short_description
-                        .as_deref()
-                        .unwrap_or(&deleted.origin_url),
-                    deleted.origin_url,
-                    deleted
-                        .metadata
-                        .map_or(String::new(), |m| format!(" ({})", m)),
-                ));
-            }
+            let resp = if deleted.is_empty() {
+                String::from("You have no pending notifications, so none were deleted.")
```

Comment:
This is misleading. You could have pending notifications, just none that matched the passed filter.

Reply:
Oops, thanks for catching that. I forgot that this runs for everything, not just glob `ack`s.